pkg/trace/trace.go

```diff
@@ -111,19 +112,19 @@ var defaultTracer tracer
 //
 // This should be called at the start of the application.
 func StartTracing(serviceName string) error {
-	if err := setDefaultTracer(); err != nil {
+	if err := setDefaultTracer(serviceName); err != nil {
```

Comment:
Once this is released, it would be nice to test this on a couple services just to validate it's 100% correct. This is a slight modification, `StartTracing` is used everywhere.